wzdx_sandbox.py

```diff
@@ -1,21 +1,88 @@
+from copy import deepcopy
 from datetime import datetime, timedelta
 import dateutil.parser
+import json
+import logging
 import requests
+import xmltodict
 
 from s3_helper import S3Helper
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)  # necessary to make sure aws is logging
 
-class WorkZoneSandbox(object):
-    def __init__(self, feed, bucket, aws_profile=None, logger=False):
-        self.prefix_template = 'state={state}/feedName={feedName}/year={year}/month={month}/'
-        self.feed = feed
+
+class ITSSandbox(object):
+    def __init__(self, bucket, aws_profile=None, logger=None):
         self.bucket = bucket
-        self.aws_profile = aws_profile
+        self.s3helper = S3Helper(aws_profile=aws_profile)
         self.print_func = print
         if logger:
             self.print_func = logger.info
 
-        self.s3helper = S3Helper(aws_profile=aws_profile)
+    def generate_fp(self, template, params):
+        return template.format(**params)
+
+
+class WorkZoneRawSandbox(ITSSandbox):
+    def __init__(self, bucket, feed=None, lambda_to_trigger=None,
+                # registry_dataset_id=None, socrata_params=None,
+                **kwargs):
+        super(WorkZoneRawSandbox, self).__init__(bucket, **kwargs)
+        self.prefix_template = 'state={state}/feedName={feedname}/year={year}/month={month}/'
+        self.feed = feed
+        self.lambda_to_trigger = lambda_to_trigger
+        # self.registry_dataset_id = registry_dataset_id
+        # self.socrata_params = socrata_params
```

Comment:
Should remove, use, or give a note about commented code

Reply:
Added additional comments about the code.